Assignment_1/Question_2.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System;
+
+namespace Assignment_1
+{
+    internal class Question_2
+    {
+        public void Question2()
+        {
+            string email = "abc@gmail.com"; 
+            int at=0;
+            int dot=0;
+            char[] E= email.ToCharArray();
+
+            for (int i = 0; i < E.Length; i++)
+            {
+                if (E[i] == '@')
+                {
+                    at += 1;
+                }else if (E[i]=='.'){
+                    dot += 1;
+                }
+                
+            }
+            if(at>0 && dot > 0)
+            {
+                Console.WriteLine("Email is valid");
```

Comment:
this will fail if there are more than 2 @... For example ab.c@gmai@l.com

Reply:
Updated code for @ symbol